sdk/core/azure-core-amqp/inc/azure/core/amqp/internal/message_sender.hpp

```diff
@@ -3,6 +3,7 @@
 
 #pragma once
 
+#include "amqp_settle_mode.hpp"
```

Comment:
```suggestion
#include "azure/core/amqp/internal/amqp_settle_mode.hpp"
```

Reply:
Same comment. There does not appear to be a consistent standard w.r.t. using the full path for include file names in the repo.

Reply:
In this repo we use full paths everywhere in `/inc`. Only the `/src` uses the relative paths.